classes/mender-sdimg.bbclass

```diff
@@ -167,15 +167,18 @@ EOF
     # Call WIC
     IMAGE_CMD_wic
 
+    wicimgname="${IMGDEPLOYDIR}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.wic"
```

Comment:
BTW. a similar patch was recently submitted for meta-raspberrypi https://lists.yoctoproject.org/pipermail/yocto/2016-September/032086.html